src/modules/users/entities/profile.entity.ts

```diff
@@ -31,13 +31,14 @@ export class Profile {
   @Column({ nullable: true, type: 'enum', enum: Gender })
   gender: Gender;
 
-  @Column()
+  @Column({ nullable: true })
   country: string;
 
-  @Column()
+  @Column({ nullable: true })
   description: string;
```

Comment:
If the column is nullable, I believe ```description``` should accept ```null``` type as well.

Reply:
Fixed